### DISCUSSION TITLE
Ability to scan shorts

### DISCUSSION THREAD

**Discussion by blahguy88** [ThioJoe/YT-Spammer-Purge#956] [Ideas]
As the spammer purge is now, I've noticed that it can't scan shorts, like this video: https://www.youtube.com/watch?v=S_CUEOBZ0P4&lc=Ugyl2hMzSu6hSHKeq6d4AaABAg.9hSvAuq3V9n9hTnlT6ohrv

Suffice to say, spam bots run rampant there too. Correct me if I'm wrong.

**Comment by Firecul** [comment]
Your example works fine for me

![image](https://user-images.githubusercontent.com/34950371/197362532-3a4eea0c-a8dc-4eaf-b692-b4fd8b366906.png)
